pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -196,6 +196,8 @@ Setting | Default values - Version 2.x | Default values - Version 3.0 | Descript
 `paths.dist.js` | `"./assets/uswds/js"` | `"./assets/uswds/js"` | Project destination for compiled JavaScript
 `paths.dist.css` | `"./assets/uswds/css"` | `"./assets/uswds/css"` | Project destination for compiled CSS
 `sprite.projectIconsOnly` | `false` | `false` | Include _only_ the icons in `paths.src.projectIcons` in the icon sprite
+`settings.compile.browserslist` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer).
+`settings.compile.sassSourcemaps` | `true` | `true` | Include sourcemap when compiling SASS to CSS.
```

Comment:
**question:** Now that we have multiple non-path settings available, should we create a separate table for them? Or rewrite the table header and intro?

Reply:
I'd keep it together for now. There aren't enough issues to warrant a completely separate table _yet_.